decidim/decidim
# Fix multitenant organizations stats cache

#### :tophat: What? Why?
When using cache with multitenant organizations, the stats are always the same for all organizations after they are cached.

#### :pushpin: Related Issues
- Fixes #9589

#### Testing
- Run two organizations with some data that differs between the organizations
- Enable cache (`bundle exec rails dev:cache`)
- Request the first organization
- Request the second organization
- See that correct stats are presented on the home page
